src/enclave/http.h

```diff
@@ -230,21 +229,25 @@ namespace enclave
 
       LOG_TRACE_FMT("recv called with {} bytes", size);
 
-      auto buf = read_all_available();
-
-      if (buf.size() == 0)
-        return;
+      while (true)
```

Comment:
This is the fix that was required on the server side: for every `tls_inbound` message sent over the ring buffer, we read as much as 4k of decrypted TLS data from mbedtls and parse it.